tests/unit/src/test/scala/tests/WorkspaceSymbolLspSuite.scala

```diff
@@ -129,7 +129,10 @@ class WorkspaceSymbolLspSuite extends BaseLspSuite("workspace-symbol") {
     } yield ()
   }
 
-  test("dependencies", withoutVirtualDocs = true) {
+  // I marked this as flaky because if I fix this locally, it ends up failing
```

Comment:
This looks like an issue when printing the results in tests. Could create an issue for a follow up?

Reply:
Yup, done.